backend/tests/conftest.py

```diff
@@ -10,7 +10,7 @@
 from conduit.profile.models import UserProfile
 
 
-from .factories import UserFactory
+from .factories import UserFactory, AdminFactory
 
```

Comment:
Pull from staging branch and test your feature again and I can merge your pull requests. Awesome work!